### PR TITLE
WL-3316 : Add Quick Links html and js to work same as My Sites

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

**Comment by nicholaswilson100** [comment]
Linked to https://github.com/ox-it/wl-reference/pull/62

**Comment by nicholaswilson100** [comment]
Not completely sure how to fix the indentation change for includeTabs.vm in IDEA. The original file has a tab on the end of a line and when I save the file it shows it as changed and without the tab.  Must be an intellij setting somewhere...

File: portal-charon/charon/src/webapp/scripts/neoscripts.js [hunk 1]
```diff
@@ -744,3 +744,36 @@ $(function() {
...
+    }
+    // finally run the inner function, first time through
+    dhtml_view_quicklinks();
+}
+
+function closeQuickLinksDrawer() {
+    jQuery('div#selectQuickLinks div').hide();
+    jQuery('div#selectQuickLinks').slideUp('fast'); // hide the box
+    removeDHTMLMask();
+    jQuery('#otherSiteTools').remove();
```
Comment (buckett) [inline portal-charon/charon/src/webapp/scripts/neoscripts.js]: I think these last 3 lines aren't needed for the Quick Links are they?

**Comment by nicholaswilson100** [comment]
Yeah good spot

**Pull request merged**